app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -130,6 +132,24 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, "/repositories/:id/export", N_("Export a repository")
+    param :id, :identifier, :required => true, :desc => N_("repository ID")
+    param :since, Date, :desc => N_("Optional ISO 8601 date of last export (ex: 2010-01-01T12:00:00), useful for exporting deltas. If not specified, a full export will occur."), :required => false
```

Comment:
That's what we do elsewhere, support the Ruby date/time supported strings and convert to what Pulp needs where appropriate.